packages/proto/.gitignore

```diff
@@ -1,2 +1,5 @@
 src/proto.js
 src/proto.d.ts
+src/proto/services
```

Comment:
```suggestion
src/proto
```

Reply:
and could remove `src/proto/streams`

Reply:
I'm not ignoring the entire **proto** folder because the mirror/sdk folders remain constant and aren't being updated. When we do update the protobufs, we update both the services and streams folders, which is why I'm excluding them individually.

Reply:
I did not see this is the actual submodule.

Reply:
Then, why do we need to ignore `/services`